utils/spacewalk-utils.spec

```diff
@@ -20,90 +21,113 @@
 %{!?pylint_check: %global pylint_check 1}
 %endif
 
-%if 0%{?fedora} || 0%{?rhel}
-%global apache_group apache
-%endif
-%if 0%{?suse_version}
-%global apache_group www
-%endif
-
 Name:           spacewalk-utils
 Version:        4.1.5
 Release:        1%{?dist}
-Summary:        Utilities that may be run against a Uyuni server
+Summary:        Utilities that may be run against a SUSE Manager/Uyuni server
```

Comment:
What's the point of adding the SUMA name here?

Reply:
This package is common for SUSE Manager and Uyuni. Basically is the same case where Red Hat had mentions to both Spacewalkd and Satellite.